templates/email/default/submit.html

```diff
@@ -57,6 +57,14 @@ <h2 style="[% h2_style %]">[% report.title | html %]</h2>
       [%~ END %][% END %]
     </p>
   [% END %]
+
+  [% IF cobrand.moniker == 'bexley' %]
```

Comment:
This template is never used by Bexley, UKCouncils.pm sets `fixmystreet.com` as an email template path above, so that one will always be used in preference.